CHANGELOG.md

```diff
@@ -1960,6 +1960,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Added `SIGINFO` in `::nix::sys::signal` for the _macos_ target as well as
   `SIGPWR` and `SIGSTKFLT` in `::nix::sys::signal` for non-_macos_ targets.
   ([#361](https://github.com/nix-rust/nix/pull/361))
+- Added `TCP_INFO` sockopt to read TCP socket information on _linux_.
+  ([#2615](https://github.com/nix-rust/nix/pull/2615))
```

Comment:
Please add the changelog entry there: https://github.com/nix-rust/nix/tree/master/changelog

Reply:
Please discard this change:)